app_integrations/apps/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Create some package level items to make this nicer to use"""
+import importlib
+import os
+
+# Import all files containing subclasses of AppIntegration, skipping the common base class
+for app_file in os.listdir(os.path.dirname(__file__)):
+    # Skip the common base file and any non-py files
+    if app_file.startswith(('__init__', 'app_base')) or not app_file.endswith('py'):
```

Comment:
@austinbyers  the logic is `not` - this skips files that _don't_ end in py

Reply:
Sorry, I wasn't clear - I meant why check for `not app_file.endswith('py')` instead of `not app_file.endswith('.py')`? Because right now, a file called `happy` would be considered a "Python file"

Reply:
ooooooh I hear you - will make the change -__-